### ISSUE TITLE
[Question] How to make ''

### ISSUE THREAD

**Issue by KsaNL** [cheat-engine/cheat-engine#2627]

**Comment by cheat-engine** [comment]
you need to pass the disableInfo structure that autoAssemble returns when registering the structure

**Comment by KsaNL** [comment]
> you need to pass the disableInfo structure that autoAssemble returns when registering the symbol

yes, I known. Is there a solution can unregister all symbol at once？
in ce7.2 notice `AA: unregistersymbol(*) - Now unregisters all symbols a script registered`, maybe it's time to let `unregisterAllSymbol` for remove all registered symbol
